Gemfile

```diff
@@ -25,30 +22,34 @@ source 'https://rubygems.org'
 gem 'bundler', supported_bundler_version
 
 group :default do
+  gem 'activesupport', '= 6.1.7.7'
   gem 'addressable', '>= 2.8.0'
-  gem 'bson', '~> 4.2.2'
+  gem 'bson', '~> 4.2.2', platform: :jruby
+  gem 'bigdecimal', '~> 3.1.7', platform: :jruby
   gem 'concurrent-ruby', '~> 1.1.4'
+  gem 'dry-cli', '~> 0.7.0'
   gem 'elasticsearch', '~> 8.13.0'
-  gem 'nokogiri', '= 1.13.10', require: false
-  gem 'json-schema'
-  gem 'activesupport', '= 6.1.7.7'
   gem 'jar-dependencies', '0.4.1'
+  gem 'json', '~> 2.7.2', platform: :jruby
+  gem 'json-schema', '~> 4.3.0'
+  gem 'nokogiri', '= 1.13.10', require: false, platform: :jruby
```

Comment:
why `require: false`? Don't we heavily need nokogiri in the crawler?

Reply:
It was already `require: false` since ent-search days and I never questioned it. I think this would be safe to remove.

Reply:
@seanstory I removed the `require: false` and also rearranged the jruby gems so it's easier to read.